site/src/components/site-menu/site-menu.tsx

```diff
@@ -27,13 +27,13 @@ export class SiteMenu {
           title: 'Installation',
           url: '/docs/getting-started/'
         },
-        {
-          title: 'PWA Elements',
-          url: '/docs/getting-started/pwa-elements/'
```

Comment:
Felt off to put this link above the Ionic one.  First you "Get Started", then go to "Ionic" page, then maybe look at the PWA elements page.